controllers/messenger.js

```diff
@@ -170,13 +170,13 @@ module.exports = () => {
     .catch(res.logger.error);
   }
 
-  function sendFeedback(res, senderId, intent, message) {
+  function sendFeedback(res, senderId, intent, rawMessage) {
+    let message = rawMessage.length >= config.maxMessageLength ? config.maxMessageLengthString : rawMessage;
```

Comment:
I think `rawMessage` is a string. So, if i'ts right. `Object.assign({}, "foo")` will return `{0:'f', 1:'o', 2: 'o'}`.